kgraphql/src/test/kotlin/com/apurebase/kgraphql/integration/DataLoaderExecutionTest.kt

```diff
@@ -75,31 +75,33 @@ class DataLoaderExecutionTest {
         }
     }
 
-    @RepeatedTest(50)
-    fun Stress_test_with_dataloaders_and_custom_superviser_jobs() {
+    @RepeatedTest(10)
```

Comment:
I don't know why this should be run 50 times. Not even sure about 10 but that's at least 80% less.